### PR TITLE
updated set-output to use new property

### PR DESCRIPTION
**Pull request by garethahealy**
$subject - think i got them all
- https://github.com/redhat-cop/github-workflows-operators/issues/19

### CODE REVIEW

File: .github/workflows/release-operator.yml [hunk 1]
```diff
@@ -88,48 +88,48 @@ jobs:
...
-            echo "::set-output name=bundle_image_registry::quay.io/$(dirname $GITHUB_REPOSITORY)"
+            echo "bundle_image_repository_name=${{ steps.set-variables.outputs.repository_name }}-bundle" >> $GITHUB_OUTPUT
+            echo "bundle_image_registry=quay.io/$(dirname $GITHUB_REPOSITORY)" >> $GITHUB_OUTPUT
 
           else
             BUNDLE_IMAGE_REPOSITORY="${{ inputs.BUNDLE_IMAGE_REPOSITORY }}"
-            echo "::set-output name=bundle_image_repository_name::${BUNDLE_IMAGE_REPOSITORY##*/}"
-            echo "::set-output name=bundle_image_registry::${BUNDLE_IMAGE_REPOSITORY%/*}"
+            echo "bundle_image_repository_name=${BUNDLE_IMAGE_REPOSITORY##*/}" >> $GITHUB_OUTPUT
+            echo "bundle_image_registry=${BUNDLE_IMAGE_REPOSITORY%/*}">> $GITHUB_OUTPUT
```
Comment (pabrahamsson) [inline .github/workflows/release-operator.yml:118]: Missing a space here
```suggestion
            echo "bundle_image_registry=${BUNDLE_IMAGE_REPOSITORY%/*}" >> $GITHUB_OUTPUT
```

**Review by pabrahamsson — CHANGES_REQUESTED** [review]
Found a missing space, otherwise looks good

**Comment by garethahealy** [comment]
@pabrahamsson ; accepted change.

**Review by pabrahamsson — APPROVED** [review]
LGTM

**Pull request merged**